### ISSUE TITLE
Título de nota em cima da linha do site

### ISSUE THREAD

**Issue by RPostalli** [scieloorg/opac#2731]
Algumas notas estão ficando desconfiguradas no site novo e estamos recebendo algumas reclamações.
O título da nota fica em cima da linha pontilhada conforme imagem abaixo. Esse problema ocorre quando a nota no XML tem a marcação de label ou quando não tem.

<img width="235" alt="image" src="https://github.com/scieloorg/opac/assets/6084959/8b7da080-494d-420d-a89b-bb9dadb80bcb">

No XML (exemplo sem label):

```
<fn-group>
			<fn fn-type="other">
				<p>
					<bold>Pareceres</bold>
				</p>
				<p>Tendo em vista o compromisso assumido por <italic>Bakhtinina</italic>. Revista de
                    Estudos do Discurso com a Ciência Aberta, a revista publica somente os pareceres
                    autorizados por todas as partes envolvidas.</p>
			</fn>
		```

**Comment by anderson-gsantos** [comment]
Abaixo uma imagem com o problema relatado em uma reclamação que recebemos:
![image](https://github.com/scieloorg/opac/assets/126583150/55bcf4ea-3ead-45a6-9a06-68e04f203928)

Também envio o XML do artigo publicado:
[1517-9702-ep-49-e252722.zip](https://github.com/scieloorg/opac/files/11850226/1517-9702-ep-49-e252722.zip)

**Comment by robertatakenaka** [comment]
@RPostalli @anderson-gsantos colocar o link dos artigos

**Comment by robertatakenaka** [comment]
@RPostalli @anderson-gsantos eu estou achando que é marcação que está fora do padrão esperado. Há 2 parágrafos nas notas de rodapé. Acho que modificar o primeiro para label e manter o segundo como parágrafo vai apresentar corretamente. Ou isso está ocorrendo em todos os XML?

**Comment by RPostalli** [comment]
> @RPostalli @anderson-gsantos eu estou achando que é marcação que está fora do padrão esperado. Há 2 parágrafos nas notas de rodapé. Acho que modificar o primeiro para label e manter o segundo como parágrafo vai apresentar corretamente. Ou isso está ocorrendo em todos os XML?

@robertatakenaka era assim que estava antes e tentando corrigir foi excluído o label oq acabou ficando pior. De qualquer forma não é qualquer nota que tem label então o site tem que ficar bem formatado com e sem label.

**Comment by robertatakenaka** [comment]
@RPostalli eu quis dizer assim:
```xml
<fn fn-type="other">
					<label>Pareceres</label>
				<p>Tendo em vista o compromisso assumido por <italic>Bakhtinina</italic>. Revista de
                    Estudos do Discurso com a Ciência Aberta, a revista publica somente os pareceres
                    autorizados por todas as partes envolvidas.</p>
			</fn>
```

**Comment by RPostalli** [comment]
> @RPostalli eu quis dizer assim:
> 
> ```
> <fn fn-type="other">
> 					<label>Pareceres</label>
> 				<p>Tendo em vista o compromisso assumido por <italic>Bakhtinina</italic>. Revista de
>                     Estudos do Discurso com a Ciência Aberta, a revista publica somente os pareceres
>                     autorizados por todas as partes envolvidas.</p>
> 			</fn>
> ```

@robertatakenaka estava assim antes e na tentativa de corrigir o problema foi excluído o label e feito reproc ficou pior. Então acho que é indiferente neste caso. Está mal formatado com e sem label.

**Comment by robertatakenaka** [comment]
@RPostalli de qq forma o padrão que tenho visto é este aí. Vou contar com o apoio do @ramoncordini @alexxxmendonca para resolver a questão da apresentação.

**Comment by RPostalli** [comment]
> @RPostalli de qq forma o padrão que tenho visto é este aí. Vou contar com o apoio do @ramoncordini @alexxxmendonca para resolver a questão da apresentação.

@robertatakenaka sim, por isso que veio com label antes. Usualmente marcam com label mesmo. Mas não é mandatório.

**Comment by ramoncordini** [comment]
@robertatakenaka @alexxxmendonca 

Analisei o problema e verifiquei que é realmente um problema de CSS. Vejam as imagens abaixo:

![Screen Shot 2023-06-27 at 09 31 29](https://github.com/scieloorg/opac/assets/5552212/e839c607-9561-4273-9309-8bc15cd5b091)
![Screen Shot 2023-06-27 at 09 29 45](https://github.com/scieloorg/opac/assets/5552212/bb408452-e382-45f4-9357-7e551ecfcbfa)

Farei os ajustes necessários.

**Comment by RPostalli** [comment]
@ramoncordini se possível dá pra corrigir o espaçamento entre as notas? @robertatakenaka PSI

A revista informou que estão grudadas e estão mesmo veja print.
<img width="248" alt="image" src="https://github.com/scieloorg/opac/assets/6084959/d4668815-336d-4af5-afcd-f842ccd3bda2">

**Comment by robertatakenaka** [comment]
@gitnnolabs  Ao testar o PR que corrige o issue, verificar se também corrige o problema mencionado no comentário acima, pois se não me engano é o mesmo estilo.

**Comment by robertatakenaka** [comment]
@ramoncordini Problema parece que foi resolvido parcialmente. 

Veja o resultado no modal. O problema está nos emails

<img width="898" alt="Captura de Tela 2023-06-30 às 13 31 44" src="https://github.com/scieloorg/opac/assets/505143/2fa7d1d6-a902-4cc7-915f-60117bd8a091">

Veja o resultado no final do texto 
<img width="1143" alt="Captura de Tela 2023-06-30 às 13 37 23" src="https://github.com/scieloorg/opac/assets/505143/38d8dd03-1427-44f1-888e-75865e07b155">

Só para te dar contexto.
Este dado idêntico é proveniente de uma "nota de rodapé" sobre os editores responsáveis.
Recentemente nos foi solicitado para dar destaque no texto completo com o link no menu esquerdo para os editores.
Por este motivos o mesmo conteúdo está tanto no modal como na página. (Fazendo literalmente um parênteses com o que já comentamos que idealmente deveríamos eliminar os modais)

Enfim, pode nos dar uma ajuda na apresentação dos emails e do texto Editora na página do artigo?

**Comment by gitnnolabs** [comment]
@scieloorg/scielo-ps-developers 

Pessoal testei localmente e também no site SciELO e esta funcionando como esperado, vejam: 

![Screenshot 2023-06-30 at 13 48 03](https://github.com/scieloorg/opac/assets/86991526/b8f3a885-121c-4236-975e-91800cb61120)
